test/s1/appframework/appframework_test.go

```diff
@@ -866,4 +866,103 @@ var _ = Describe("s1appfw test", func() {
+			*/
+
+			//################## SETUP ####################
+
+			// Creating a list of apps to be installed on both standalone
+			appList1 := append(appListV1, testenv.RestartNeededApps[len(testenv.RestartNeededApps)/2:]...)
+			appList2 := append(appListV1, testenv.RestartNeededApps[:len(testenv.RestartNeededApps)/2]...)
+			appVersion := "V1"
+
+			// Download apps from S3
+			testenvInstance.Log.Info("Download apps from S3 for this test")
+			appFileList := testenv.GetAppFileList(testenv.RestartNeededApps)
+			err := testenv.DownloadFilesFromS3(testDataS3Bucket, s3AppDirV1, downloadDirV1, appFileList)
+			Expect(err).To(Succeed(), "Unable to download apps files")
+
+			// Upload apps to S3 for first Standalone
+			testenvInstance.Log.Info("Upload bigger amount of apps to S3 for this test")
+			appFileListStandalone1 := testenv.GetAppFileList(appList1)
+			uploadedFiles, err := testenv.UploadFilesToS3(testS3Bucket, s3TestDir, appFileListStandalone1, downloadDirV1)
+			Expect(err).To(Succeed(), "Unable to upload apps to S3 test directory")
+			uploadedApps = append(uploadedApps, uploadedFiles...)
+
+			// Upload apps to S3 for second Standalone
+			testenvInstance.Log.Info("Upload bigger amount of apps to S3 for this test")
+			s3TestDirStandalone2 := "s1appfw-2-" + testenv.RandomDNSName(4)
+			appFileListStandalone2 := testenv.GetAppFileList(appList2)
+			uploadedFiles, err = testenv.UploadFilesToS3(testS3Bucket, s3TestDirStandalone2, appFileListStandalone2, downloadDirV1)
+			Expect(err).To(Succeed(), "Unable to upload apps to S3 test directory")
+			uploadedApps = append(uploadedApps, uploadedFiles...)
+
+			// Create App framework Spec
+			appSourceName = "appframework-" + enterpriseApi.ScopeLocal + testenv.RandomDNSName(3)
+			appFrameworkSpec := testenv.GenerateAppFrameworkSpec(testenvInstance, appSourceVolumeName, enterpriseApi.ScopeLocal, appSourceName, s3TestDir, 60)
+			spec := enterpriseApi.StandaloneSpec{
+				CommonSplunkSpec: enterpriseApi.CommonSplunkSpec{
+					Spec: splcommon.Spec{
+						ImagePullPolicy: "Always",
+					},
+					Volumes: []corev1.Volume{},
+				},
+				AppFrameworkConfig: appFrameworkSpec,
+			}
+
+			// Create App framework Spec
+			appSourceNameStandalone2 := "appframework-2-" + enterpriseApi.ScopeLocal + testenv.RandomDNSName(3)
+			appSourceVolumeNameStandalone2 := "appframework-test-volume-2-" + testenv.RandomDNSName(3)
+			appFrameworkSpecStandalone2 := testenv.GenerateAppFrameworkSpec(testenvInstance, appSourceVolumeNameStandalone2, enterpriseApi.ScopeLocal, appSourceNameStandalone2, s3TestDirStandalone2, 60)
+			specStandalone2 := enterpriseApi.StandaloneSpec{
+				CommonSplunkSpec: enterpriseApi.CommonSplunkSpec{
+					Spec: splcommon.Spec{
+						ImagePullPolicy: "Always",
+					},
+					Volumes: []corev1.Volume{},
+				},
+				AppFrameworkConfig: appFrameworkSpecStandalone2,
+			}
+
+			// Deploy Standalone
+			testenvInstance.Log.Info("Deploy Standalone")
```

Comment:
Nit: may need to have 2 lines as below for clarity:
testenvInstance.Log.Info("Deploy 1st Standalone")
testenvInstance.Log.Info("Deploy 2nd Standalone")